app/routes/changes-enquiries/death/functions.js

```diff
@@ -34,9 +34,10 @@ const [CANNOT_CALCULATE, OVERPAYMENT, ARREARS, NOTHING_OWED, DEATH_NOT_VERIFIED]
 
 function getAddDateDeath(req, res) {
   const { awardDetails } = req.session;
+  const keyDetails = keyDetailsHelper.formatter(awardDetails);
   const details = dataStore.get(req, 'date-of-death', 'death');
   res.render('pages/changes-enquiries/death/enter-date', {
-    awardDetails,
+    keyDetails,
```

Comment:
Remember you talking about this but wasn't quite sure what you meant. But we don't actually need to do this as keyDetails is passed to the locals and to the views via middleware. If this exists in other functions you can remove it. The middleware was an afterthought so went through all the code to remove `keyDetails` but might have missed a few

Reply:
removed